### PR TITLE
Bootstrap `web-ui` `README.md`

### PR DESCRIPTION
**Pull request by shandysulen**
Provide project-specific guidance in `packages/web-ui/README.md` to run `web-ui` locally.

### CODE REVIEW

File: packages/web-ui/README.md [hunk 1]
```diff
@@ -1,46 +1,33 @@
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+`packages/abi-to-sol` hasn't been built. Go to `packages/abi-to-sol` and run `yarn prepare`.
```
Comment (gnidan) [inline packages/web-ui/README.md:33]: Hm kind of annoying that `yarn` doesn't automatically build everything in the repo root. Nonetheless, good to have these instructions here!

**Review by gnidan — APPROVED** [review]
This looks good, thanks for putting these instructions in the README!

Note that my CI build here requires adding every PR to the CHANGELOG. You mind doing that and then I'll merge this?

File: packages/web-ui/README.md (hunk 1 above)
Comment (shandysulen) [reply packages/web-ui/README.md:33]: CHANGELOG updated!

Agreed, I would think that running `yarn` in monorepo root would still trigger lifecycle scripts, like `prepare`, in individual packages. There's probably a nice solution out there -- we can update the README once we find it.

**Comment by gnidan** [comment]
Thanks again!

**Pull request merged**